Signal/src/network/PushManager.m

```diff
@@ -216,9 +216,6 @@ - (void)application:(UIApplication *)application
             return;
         }
 
-        TSContactThread *thread = [TSContactThread getOrCreateThreadWithContactId:recipientId];
-        [MarkIdentityAsSeenJob runWithThread:thread];
```

Comment:
1. We should either do something like`OWSIdentityManager.shared().setVerificationState(.default, identityKey: noLongerVerifiedIdentity.identityKey, recipientId: noLongerVerifiedIdentity.recipientId, sendSyncMessage: true)

2. Or maybe just remove the option to call back from the lockscreen in this case, since we don't have a reference to the identity here.

I'm leaning towards 2.